test/hessian.jl

```diff
@@ -70,13 +68,13 @@ include("testcases.jl")
         @testset "solve_ΩplusK_split <=> solve_ΩplusK" begin
             scfres = self_consistent_field(basis; tol=1e-10)
             δψ1 = solve_ΩplusK_split(scfres, rhs).δψ
+            # TODO: No idea why Hessian mandatory. Same issue even without using setup…
```

Comment:
I don't get this comment ?

Reply:
Ok, let's finish up the rest and than I'll take a look at that and see if I can figure it out.